python/python/tests/test_integration.py

```diff
@@ -7,6 +7,20 @@
 import pytest
 
 
+@pytest.mark.skip(reason="Unsigned integers not available for pushdown yet")
+def test_duckdb_filter_on_rowid(tmp_path):
+    tab = pa.table({"a": [1, 2, 3], "b": [4, 5, 6]})
+    ds = lance.write_dataset(tab, str(tmp_path))
+    ds = lance.dataset(str(tmp_path), default_scan_options={"with_row_id": True})
```

Comment:
Unimportant, but these should take paths as-is:
```suggestion
    ds = lance.write_dataset(tab, tmp_path)
    ds = lance.dataset(tmp_path, default_scan_options={"with_row_id": True})
```